powersync-ja/powersync-js
# feat: add electron web demo

## Description
Add a demo showcasing how Electron can be used with a rendered web application.

## Work Done
* Included electron demo application
* Updated root README

## How to test
* Run `cd demos/example-electron`
* Run `pnpm start`
* Confirm app works as expected when offline or online and switching between offline to online

## Screenshot
<img width="1726" alt="image" src="https://github.com/powersync-ja/powersync-js/assets/46312751/524b9846-cdd6-4d6d-a5a4-cea224db6acb">
